dev-infrastructure/templates/mgmt-cluster.bicep

```diff
@@ -107,6 +107,11 @@ module mgmtCluster '../modules/aks-cluster-base.bicep' = {
         namespace: 'maestro'
         serviceAccountName: 'maestro'
       }
+      package_operator: {
+        uamiName: 'package-operator'
+        namespace: 'package-operator-system'
```

Comment:
i've recently started to refactor services to have config.yaml parameters for MI names, namespaces names and service account names.
https://github.com/Azure/ARO-HCP/pull/1102